examples/evm-to-evm-erc20-contract-call/src/transfer.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  Config,
+  EvmResource,
+  getSygmaScanLink,
+  type Eip1193Provider,
+} from "@buildwithsygma/core";
+import {
+  createFungibleAssetTransfer,
+  FungibleTransferParams,
+} from "@buildwithsygma/evm";
+import dotenv from "dotenv";
+import { Wallet, ethers, providers } from "ethers";
+import Web3HttpProvider from "web3-providers-http";
+import { getContractAddress, getContractInterface } from "./contracts";
+
+dotenv.config();
+
+const privateKey = process.env.PRIVATE_KEY;
+
+if (!privateKey) {
+  throw new Error("Missing environment variable: PRIVATE_KEY");
+}
+
+const SEPOLIA_CHAIN_ID = 11155111;
+const BASE_SEPOLIA_CHAIN_ID = 84532;
+const RESOURCE_ID =
+  "0x0000000000000000000000000000000000000000000000000000000000001200";
+const SEPOLIA_RPC_URL =
+  process.env.SEPOLIA_RPC_URL || "https://ethereum-sepolia-rpc.publicnode.com";
+
+const explorerUrls: Record<number, string> = {
+  [SEPOLIA_CHAIN_ID]: "https://sepolia.etherscan.io",
+};
+const getTxExplorerUrl = (params: {
+  txHash: string;
+  chainId: number;
+}): string => `${explorerUrls[params.chainId]}/tx/${params.txHash}`;
+
+export async function erc20Transfer(): Promise<void> {
+  const web3Provider = new Web3HttpProvider(SEPOLIA_RPC_URL);
+  const ethersWeb3Provider = new providers.Web3Provider(web3Provider);
+  const wallet = new Wallet(privateKey ?? "", ethersWeb3Provider);
+  const sourceAddress = await wallet.getAddress();
+  const destinationAddress = await wallet.getAddress();
+  const contract = "sprinterNameService";
+
+  const config = new Config();
+  await config.init(process.env.SYGMA_ENV);
+  const resource = config
```

Comment:
from config we have `getResources` method which gives you the resources from domain